src/main/java/lotto/Number.java

```diff
@@ -0,0 +1,55 @@
+package lotto;
+
+import java.util.Objects;
+
+public class Number implements Comparable {
+
+    private final int value;
+
+    private Number(int value) {
+        this.value = value;
+    }
+
+    public static Number newNumber(int value) {
+        if (value > LottoTotalNumberUtils.LOTTO_NUMBER_MAXIMUM) {
+            throw new IllegalArgumentException("로또 번호는 45보다 클 수 없습니다.");
```

Comment:
```suggestion
            throw new IllegalArgumentException("로또 번호는 " + LOTTO_NUMBER_MAXIMUM + "보다 클 수 없습니다.");
```
상수를 사용 해보면 좋을것 같아요!
아래 코드도 적용해 볼 수 있겠네요 😄